core-container/entry.sh

```diff
@@ -242,51 +219,66 @@ function run_kcp() {
 
 function run_init() {
     echo "--< Starting init >--"
-    wait_kcp_ready
-    set_provider_adapters
-    kubestellar init --local-kcp false --ensure-imw $ENSURE_IMW --ensure-wmw $ENSURE_WMW
-    kubectl ws root
+    create_spaceprovider_object
+    kubestellar init --ensure-imw $ENSURE_IMW --ensure-wmw $ENSURE_WMW
     touch ready
     echo "***READY***"
     sleep infinity
 }
 
-
 function run_mailbox_controller() {
     echo "--< Starting mailbox-controller >--"
     wait-kubestellar-ready
+    # TODO: remove the kcp dependency in PR3
+    get_kcp_kubeconfig
+    KUBECONFIG=$kcp_kubeconfig
     kubectl ws root:espw
     if ! mailbox-controller -v=${VERBOSITY} ; then
         echoerr "unable to start mailbox-controller!"
         exit 1
     fi
 }
 
-
 function run_where_resolver() {
     echo "--< Starting where-resolver >--"
     wait-kubestellar-ready
+    # TODO: remove the kcp dependency in PR3
+    get_kcp_kubeconfig
+    KUBECONFIG=$kcp_kubeconfig
     kubectl ws root:espw
     if ! kubestellar-where-resolver -v ${VERBOSITY} ; then
         echoerr "unable to start kubestellar-where-resolver!"
         exit 1
     fi
 }
 
-
 function run_placement_translator() {
     echo "--< Starting placement-translator >--"
     wait-kubestellar-ready
+    # TODO: remove the kcp dependency in PR3
+    get_kcp_kubeconfig
+    KUBECONFIG=$kcp_kubeconfig
     kubectl ws root:espw
     if ! placement-translator --allclusters-context  "system:admin" -v=${VERBOSITY} ; then
         echoerr "unable to start mailbox-controller!"
         exit 1
     fi
 }
 
+function set_host_kubeconfig() {
+    kubectl --kubeconfig $host_kubeconfig config set-cluster sm-mgt --server="https://${KUBERNETES_SERVICE_HOST}:${KUBERNETES_SERVICE_PORT}" --certificate-authority=/var/run/secrets/kubernetes.io/serviceaccount/ca.crt
```

Comment:
What does the "m" in the first part of "sm-mgt" stand for?   If it is "management" or "manager" then this name looks redundant.